projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-bar/index.tsx

```diff
@@ -100,6 +100,11 @@ export default function AiAssistantBar( {
 	const { removeNotice } = useDispatch( noticesStore );
 
 	const onSend = useCallback( () => {
+		// Do not send the request if the input value is empty.
+		if ( ! inputValue?.length ) {
```

Comment:
What about a min length value? Like... 7 (the length of "contact" word). That should avoid getting nonsense prompts like single letters or numbers, or people trying to greet the AI and getting a form in response.